12.glTF/main.cpp

```diff
@@ -448,12 +454,19 @@ class GLTFApp : public ApplicationBase
 			}
 			// transfer compressed aabbs to the GPU
 			{
-				aabbBinding = {0ull,utilities->createFilledDeviceLocalBufferOnDedMem(transferUpQueue,sizeof(CompressedAABB)*aabbPool.size(),aabbPool.data())};
+				IGPUBuffer::SCreationParams params1 = {};
```

Comment:
Please put some minimum effort into naming things instead of counting things, for example `aabbBufferParams` instead of `params1` and `iotaBufferParams` instead of `params2`
At the end of the day we don't write these to just work, we want to be understandable for new users